app/views/transferConfirmation.scala.html

```diff
@@ -0,0 +1,29 @@
+@()(implicit messages: Messages)
+@main(Messages("transferConfirmation.header")) {
+    @defining(play.core.PlayVersion.current) { version =>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+                @progressIndicator(Messages("transferConfirmation.progress"))
+                <div class="govuk-panel govuk-panel--confirmation">
+                    <h1 class="govuk-panel__title">
+                        @Messages("transferConfirmation.transferComplete")
+                    </h1>
+                    <div class="govuk-panel__body">
+                        @Messages("transferConfirmation.yourTransferIsComplete")
+                    </div>
+                </div>
+                <div>
+                    <!-- Dashboard -->
+                    <button class="govuk-button" data-module="govuk-button">
```

Comment:
Should the `button />` be an `<a />` link back to the Dashboard page? I don't think we will be submitting anything from this page.

Reply:
Yes, I tried with `<a>` but the CSS looked wrong, but I'll remove the button so that the person that picks up the task to implement this page can do it properly.